dash-pipeline/SAI/sai_api_gen.py

```diff
@@ -361,8 +360,11 @@ def write_sai_impl_files(sai_api):
     env.add_extension('jinja2.ext.loopcontrols')
     env.add_extension('jinja2.ext.do')
     sai_impl_tm = env.get_template('/templates/saiapi.cpp.j2')
-    sai_impl_str = sai_impl_tm.render(tables = sai_api[TABLES_TAG], app_name = sai_api['app_name'])
-
+    if "dash" in sai_api['app_name']:
```

Comment:
do we need this generator change at all, since in the end we  are using the existing header file from inc folder

Reply:
Yes, we need this change.
The "if" statement is the differentiator for generating the header files.
if "dash" keyword is used as a part of table name, then the header file is generated else the header file is not generated